indico/modules/events/models/roles.py

```diff
@@ -21,6 +21,7 @@ class EventRole(db.Model):
 
     is_group = False
     is_event_role = True
+    is_category_role = False
```

Comment:
let's add this to the other objects that have `is_event_role = False` as well.

right now neither of them are being read, but that way we are consistent (and either keep them or remove both of them later)